src/classad/collection.cpp

```diff
@@ -441,16 +436,13 @@ TruncateLog( )
         CondorErrno = ERR_LOG_OPEN_FAILED;
         CondorErrMsg = "when truncating log, failed to open " + tmpLogFileName
             + " errno=";
-        sprintf( buf, "%d", errno );
-        CondorErrMsg += string( buf );
+        CondorErrMsg += std::sto_string( errno );
```

Comment:
```suggestion
        CondorErrMsg += std::sto_string( errno );
```

Reply:
Yeah. Forgot to 'git add' the fix before committing and pushing.